Bottom: 10498e1dc1bff43c047763e4fd6598c129dba578
Top:    9c29b027f1f7b4996c953fa89b9674ef1fb5329c
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2010-11-22 08:15:57 +0100

ignore backups


---

diff --git a/bytecode/.gitignore b/bytecode/.gitignore
index 9a033b2..38ed95c 100644
--- a/bytecode/.gitignore
+++ b/bytecode/.gitignore
@@ -7,3 +7,4 @@
 /.settings
 /lib_managed
 /project/boot
+*~
